frontend/src/i18n.js

```diff
@@ -32,22 +33,36 @@ function setupI18n(options = {}) {
     if (!hasLoadedLocale(locale_)) {
         isDownloading.set(true)
 
-        const messagesFileUrl = MESSAGE_FILE_URL_TEMPLATE.replace(
+        const messagesFileUrlBase = MESSAGE_FILE_URL_BASE.replace(
             '{locale}',
             locale_,
         )
 
         // Download translation file for given locale/language
-        return loadJson(messagesFileUrl).then(messages => {
-            _activeLocale = locale_
+        loadJson(messagesFileUrlBase).then(messages => {
+            //console.log("Load base file: " + messagesFileUrlBase)
 
-            // Configure svelte-i18n to use the locale
+            // Add messages for locale
             addMessages(locale_, messages)
+        })
+
+        const messageFileUrlAdd = MESSAGE_FILE_URL_ADD.replace(
+            '{locale}',
+            locale_,
+        )
+
+        // Download additional translation file (default/friendly) for given locale/language
+        loadJson(messageFileUrlAdd).then(messages => {
```

Comment:
So these are promises which means you will have to nest this additional loadJson call into the first one and move the isDownloading.set(false) into this one. e.g.

```
loadJson().then(messages => { loadJson().then(m => { isDownloading.set(false) }) })
```

Reply:
Done.

Reply:
Thanks!

I'll pull this down and test today, appreciate all your hard work ;)